docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -28,38 +28,40 @@ module.exports = function (grunt) {
         files: ['src/**/*.{scss,sass}'],
         tasks: ['dart-sass:dist'],
         options: {
-          interrupt: true
-        }
-      }
+          interrupt: true,
+        },
+      },
     },
     run: {
       stylelint: {
-        exec: 'npm run lint:sass'
-      }
+        exec: 'npm run lint:sass',
+      },
     },
     postcss: {
       options: {
         map: false, // inline sourcemaps
         processors: [
           require('autoprefixer')({
-            grid: true
-          }) // add vendor prefixes
-        ]
+            grid: true,
+          }), // add vendor prefixes
+        ],
       },
       dist: {
         src: [
           '../humsci_colorful/**/*.css',
           '../humsci_airy/**/*.css',
-          '../humsci_traditional/**/*.css'
-        ]
-      }
+          '../humsci_traditional/**/*.css',
+        ],
+      },
     },
     'dart-sass': {
       options: {
         implementation: sass,
         sourceMap: false,
         outputStyle: 'compressed',
-        includePaths: getIncludeFiles()
+        importer: tilde_importer,
+        includePaths: getIncludeFiles(),
+        quietDeps: true,
```

Comment:
This line is preventing deprecation warnings. Otherwise, we see 15-25 warnings every time we compile sass. They are: 
```
Deprecation Warning: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($size, $columns) or calc($size / $columns)
```